### PR TITLE
Keyboard download availability API

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

**Review by jahorton — APPROVED** [review]
LGTM.

File: _tools/syncfiles.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 api/.htaccess
 api/versioninfo.php
 api/historydata.php
+api/keyboard.php
+api/util.php
```
Comment (darcywong00) [inline _tools/syncfiles.txt:5]: This impacts the [Files to Upload](https://github.com/keymanapp/downloads.keyman.com#files-to-upload) section of README.md

**Review by darcywong00 — CHANGES_REQUESTED** [review]
1. Updates to README.md
2. Is there going to be a folder for https://downloads.keyman.com/keyboard/ ? If so, that'll need .htaccess, right?

**Comment by mcdurdin** [comment]
1. Resolved.
2. Yes, added the .htaccess (for `Options +Indexes` support) to keyboards/

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**